build-tools/packages/build-tools/src/test/data/tabs/_package.json

```diff
@@ -0,0 +1,151 @@
+{
```

Comment:
I think having smaller/simpler test data would be better. One thing that annoys me with test data like this is when I search the codebase for real stuff, sometimes I find test files and think they are out of date. SInce you are just using this for indent detection, the file could just be a couple lines of clearly dummy data.

Reply:
Fair point. I'll simplify this.